dotcom-rendering/src/web/components/ShowMore.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { userEvent, within } from '@storybook/testing-library';
+import fetchMock from 'fetch-mock';
+import { trails } from '../../../fixtures/manual/show-more-trails';
+import { ShowMore } from './ShowMore.importable';
+
+/**
+ * Clicks the 'show more' button so that Chromatic can capture it the component
+ * in its 'open' state.
+ */
+const play = ({ canvasElement }: { canvasElement: HTMLElement }) => {
```

Comment:
shout out to @AshCorr for suggesting this!

Given that it required installing a new package (`@storybook/testing-library`) I'm guessing we don't use this pattern already. It seems to work fine with Chromatic, but if people have reservations about introducing  it here then I'm happy to split it off into a  separate PR, as it's not part of the core of this particular issue and does have  broader  implications.

Reply:
I think this is a very nice feature, but in the spirit of keeping PRs as atomic as possible it would be better to extract this change into its own.

Reply:
I've removed the commit from this branch, and put it on a new one; I'll open a PR once this is merged into `main`:
https://github.com/guardian/dotcom-rendering/compare/ap/show-more-island...pf/story-interactions-for-show-more?expand=1